src/PagesPlugin.js

```diff
@@ -3,21 +3,22 @@ import cheerio from 'cheerio';
 import loaderUtils from 'loader-utils';
 
 type RenderResult = {
-  markup: string
-}
+  markup: string,
+};
+
+type OutputResult = {
+  markup: string,
+  path: string,
+};
```

Comment:
we should include the output filename here as well to allow renaming the file before it is emitted